wordpress-mobile/WordPress-iOS
# Blaze Manage Campaigns: Campaign cell UI

Part of #20762 

## Description
Adds the Blaze campaign cell.

> **Note**
> Currently the campaigns screen displays mocked campaign data. Fetching actual campaigns will be addressed in a future PR.

<img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/54b326bf-1455-4954-af49-2a5ca7dcbade" width=200>

## How to test
1. Change L58 in DashboardCard.swift to return `DashboardBlazeCampaignCardCell.self`
2. Run the Jetpack app
3. Switch to a site that has Blaze enabled
4. Tap on the Blaze dashboard card
5. ✅ Verify: the campaign cell UI matches the Figma specs (ref: 1Osrn9gb2xLaKG6lIft3WC-fi-1461_9777)

## Regression Notes
1. Potential unintended areas of impact
n/a

6. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

7. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:


UI Changes testing checklist:
